content/docs/rfcs/36/README.md

```diff
@@ -106,6 +106,105 @@ Fields:
 - `data`: Decrypted message payload base64 encoded,
 - `padding`: Padding base64 encoded.
 
+### `FilterSubscription` type
+
+The criteria to create subscription to a light node in JSON Format:
+
+```ts
+{
+    contentFilters: ContentFilter[];
+    topic: string?;
```

Comment:
```suggestion
    pubsubTopic: string?;
```

As we have two types of topics (pubsub, content), I think it makes it easier to new developers to always qualify the topic.